quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testStartHelpAll.unix.approved.txt

```diff
@@ -93,6 +93,9 @@ Storage (Experimental):
                      Experimental: Sets the port of the Infinispan server.
 --storage-hotrod-username <username>
                      Experimental: Sets the username of the Infinispan user.
+--storage-jpa-db <type>
+                     Experimental: The database vendor for jpa map storage. Possible values are:
+                       postgres, cockroach. Default: postgresql.
```

Comment:
Like this, right?

```suggestion
                       postgresql, cockroach. Default: postgresql.
```

Reply:
`postgres` should IMO remain, but default value should be updated in the next commit.